modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/search/ConfigurationIndexingExtender.java

```diff
@@ -45,6 +45,10 @@
 
 	@Activate
 	protected void activate(BundleContext bundleContext) {
+		if (!_clusterMasterExecutor.isMaster()) {
```

Comment:
@tinatian I don't think this is right. The master role may change at runtime, and you won't get a chance to reopen the tracker, right?

Reply:
@shuyangzhou Ah! You are right! I will send you a new pull for this.